storage/tree/node_id2.go

```diff
@@ -101,15 +118,9 @@ func (n NodeID2) String() string {
 	return fmt.Sprintf("[%s%0*b]", path, n.bits, n.last>>(8-n.bits))
 }
 
-// split returns decomposition of a NodeID2 with the given number of bits.
-//
-// The first int of the returned triple is the number of full bytes stored in
-// the dynamically allocated part. The second one is the number of bits in the
-// tail byte (between 1 and 8). The third value is a mask with the
-// corresponding number of higher bits set.
-func split(bits uint) (uint, uint8, byte) {
-	bytes := (bits - 1) / 8
-	tailBits := uint8(1 + (bits-1)%8)
-	mask := ^byte(1<<(8-tailBits) - 1)
-	return bytes, tailBits, mask
+// split returns the decomposition of a NodeID2 with the given number of bits.
+// The first int returned is the number of full bytes stored in the dynamically
+// allocated part. The second one is the number of bits in the tail byte.
+func split(bits uint) (uint, uint8) {
```

Comment:
For increased clarity consider naming the return variables 
```suggestion
func split(bits uint) (bytes uint, tailBits uint8) {
```

Reply:
Good one, I will add it.

Reply:
Done.